site/source/user-manual/connecting/connecting-tor/tor-os/tor-windows.rst

```diff
@@ -25,8 +25,6 @@ Running Tor on Windows
 
     Note Tor destination folder when installing
```

Comment:
This is where we need to say where to put it - probably give the program files location, noting that if they change this location, it must still be in the system (and not user) space